db/re/item_db.conf

```diff
@@ -151568,4 +151929,664 @@ item_db: (
+	Name: "ArchLine7Lv"
+	Type: 6
+	Loc: 1023
+	Script: <"
+		bonus bLongAtkRate, 28;
+		bonus bAspdRate, 7;
+	">
+},
+{
+	Id: 29098
+	AegisName: "ArchLine8Lv"
+	Name: "ArchLine8Lv"
+	Type: 6
+	Loc: 1023
+	Script: <"
+		bonus bLongAtkRate, 32;
+		bonus bAspdRate, 8;
+	">
+},
+{
+	Id: 29099
+	AegisName: "ArchLine9Lv"
+	Name: "ArchLine9Lv"
+	Type: 6
+	Loc: 1023
+	Script: <"
+		bonus bLongAtkRate, 36;
+		bonus bAspdRate, 9;
+	">
+},
+{
+	Id: 29100
+	AegisName: "ArchLine10Lv"
+	Name: "ArchLine10Lv"
+	Type: 6
+	Loc: 1023
+	Script: <"
+		bonus bLongAtkRate, 44;
+		bonus bAspdRate, 10;
+	">
+},
+{
+	Id: 29101
+	AegisName: "Steel1Lv"
+	Name: "Steel1Lv"
+	Type: 6
+	Loc: 1023
+	Script: <"
+		bonus bMaxHPrate, 5;
+		bonus bDef, 15;
+	">
+},
+{
+	Id: 29102
+	AegisName: "Steel2Lv"
+	Name: "Steel2Lv"
+	Type: 6
+	Loc: 1023
+	Script: <"
+		bonus bMaxHPrate, 10;
+		bonus bDef, 30;
+	">
+},
+{
+	Id: 29103
+	AegisName: "Steel3Lv"
+	Name: "Steel3Lv"
+	Type: 6
+	Loc: 1023
+	Script: <"
+		bonus bMaxHPrate, 15;
+		bonus bDef, 45;
+	">
+},
+{
+	Id: 29104
+	AegisName: "Steel4Lv"
+	Name: "Steel4Lv"
+	Type: 6
+	Loc: 1023
+	Script: <"
+		bonus bMaxHPrate, 20;
+		bonus bDef, 60;
+	">
+},
+{
+	Id: 29105
+	AegisName: "Steel5Lv"
+	Name: "Steel5Lv"
+	Type: 6
+	Loc: 1023
+	Script: <"
+		bonus bMaxHPrate, 25;
+		bonus bDef, 75;
+	">
+},
+{
+	Id: 29106
+	AegisName: "Steel6Lv"
+	Name: "Steel6Lv"
+	Type: 6
+	Loc: 1023
+	Script: <"
+		bonus bMaxHPrate, 30;
+		bonus bDef, 90;
+	">
+},
+{
+	Id: 29107
+	AegisName: "Steel7Lv"
+	Name: "Steel7Lv"
+	Type: 6
+	Loc: 1023
+	Script: <"
+		bonus bMaxHPrate, 35;
+		bonus bDef, 105;
+	">
+},
+{
+	Id: 29108
+	AegisName: "Steel8Lv"
+	Name: "Steel8Lv"
+	Type: 6
+	Loc: 1023
+	Script: <"
+		bonus bMaxHPrate, 40;
+		bonus bDef, 120;
+	">
+},
+{
+	Id: 29109
+	AegisName: "Steel9Lv"
+	Name: "Steel9Lv"
+	Type: 6
+	Loc: 1023
+	Script: <"
+		bonus bMaxHPrate, 45;
+		bonus bDef, 135;
+	">
+},
+{
+	Id: 29110
+	AegisName: "Steel10Lv"
+	Name: "Steel10Lv"
+	Type: 6
+	Loc: 1023
+	Script: <"
+		bonus bMaxHPrate, 55;
+		bonus bDef, 150;
+	">
+},
+{
+	Id: 29111
+	AegisName: "Mercy1Lv"
+	Name: "Mercy1Lv"
+	Type: 6
+	Loc: 1023
+	Script: <"
+		bonus bHealPower, 5;
+		bonus bMdef, 3;
+	">
+},
+{
+	Id: 29112
+	AegisName: "Mercy2Lv"
+	Name: "Mercy2Lv"
+	Type: 6
+	Loc: 1023
+	Script: <"
+		bonus bHealPower, 10;
+		bonus bMdef, 6;
+	">
+},
+{
+	Id: 29113
+	AegisName: "Mercy3Lv"
+	Name: "Mercy3Lv"
+	Type: 6
+	Loc: 1023
+	Script: <"
+		bonus bHealPower, 15;
+		bonus bMdef, 9;
+	">
+},
+{
+	Id: 29114
+	AegisName: "Mercy4Lv"
+	Name: "Mercy4Lv"
+	Type: 6
+	Loc: 1023
+	Script: <"
+		bonus bHealPower, 20;
+		bonus bMdef, 12;
+	">
+},
+{
+	Id: 29115
+	AegisName: "Mercy5Lv"
+	Name: "Mercy5Lv"
+	Type: 6
+	Loc: 1023
+	Script: <"
+		bonus bHealPower, 25;
+		bonus bMdef, 15;
+	">
+},
+{
+	Id: 29116
+	AegisName: "Mercy6Lv"
+	Name: "Mercy6Lv"
+	Type: 6
+	Loc: 1023
+	Script: <"
+		bonus bHealPower, 30;
+		bonus bMdef, 18;
+	">
+},
+{
+	Id: 29117
+	AegisName: "Mercy7Lv"
+	Name: "Mercy7Lv"
```

Comment:
Name field is the user-visible name. It should match the official name used in the client where possible.

AegisName should be an identifier usable in the scripting language (so only letters, underscores, numbers).